update-dependencies/Program.cs

```diff
@@ -109,7 +108,7 @@ private static IDependencyUpdater CreateDependencyUpdater(string path)
             {
                 Path = path,
                 BuildInfoName = "Cli",
-                Regex = new Regex($@"ENV DOTNET_SDK_VERSION (?<version>[^\r\n]*)"),
+                Regex = new Regex($@"ENV DOTNET_SDK_VERSION [\d.]*-(?<version>{s_config.CliReleaseMoniker}-[^\r\n]*)"),
```

Comment:
Should this be `\d\.` to only match digits and the '.' character rather than matching all characters with the unescaped `.`?